### ISSUE TITLE
TypeError: href is undefined

### ISSUE THREAD

**Issue by ctron** [AxaGuilDEv/react-oidc#664]
### Issue and Steps to Reproduce

TypeError: href is undefined

### Versions

3.1.7

### Screenshots

![image](https://user-images.githubusercontent.com/202474/136813775-4c8dd9e0-8579-4100-a02c-3fdb0d65e3b4.png)

### Additional Details

~~~
  "dependencies": {
    "@axa-fr/react-oidc-context": "^3.1.7",
    "oidc-client": "^1.11.5",
    "@patternfly/react-core": "^4.135.2",
    "@patternfly/react-table": "^4.31.3",
    "react": "^16.14.0",
    "react-dom": "^16.14.0",
    "react-router": "^5.1.2",
    "react-router-dom": "^5.1.2",
    "web-vitals": "^1.0.1"
  },
~~~

**Comment by guillaume-chervet** [comment]
Did you try to run the demo from this Repository ?

**Comment by guillaume-chervet** [comment]
This will be fixed in the futur v4.

**Comment by guillaume-chervet** [comment]
Demo is simplified and fixed now https://black-rock-0dc6b0d03.1.azurestaticapps.net/

**Comment by guillaume-chervet** [comment]
Demo is deploy now on each pullrequest on azure static webapp: https://black-rock-0dc6b0d03.1.azurestaticapps.net/

Thank you very much @ctron for the feeback. I close the issue feel free to reopen it you have any problem.
